src/graphql/courses.ts

```diff
@@ -1,8 +1,8 @@
 import { gql } from "../__generated__/gql";
 
 export const GET_COURSES = gql(`
-  query Query($orderBy: [CourseOrderByInput!]!, $take: Int) {
-    courses(orderBy: $orderBy, take: $take) {
+  query getCourse($where: CourseWhereInput!) {
```

Comment:
@G0LD3N-c0de Sorry I should have made this clearer while we were pairing on this; we still need the other query for GET_COURSES because it's used on the main page for listing the course cards-- see `/src/routes/home/courses.tsx`. What I was trying to say is that you can add a second query to the same file called GET_COURSE that will look like what you have here, but leave GET_COURSES untouched